d2l-button-toggle-group.js

```diff
@@ -0,0 +1,156 @@
+import {html, PolymerElement} from '@polymer/polymer/polymer-element.js';
+import './d2l-button-toggle.js';
+
+/**
+ * @customElement
+ * @polymer
+ */
+export class D2LButtonToggleGroup extends PolymerElement {
+	static get is() { return 'd2l-button-toggle-group'; }
+	static get template() {
+		return html`
+			<style>
+				div ::slotted(.d2l-button-toggle-grouped) {
+					--d2l-button-toggle-border-radius: 0;
+					--d2l-button-toggle-border-width: 1px 1px 1px 0;
+				}
+				:host([orientation="vertical"]) div ::slotted(.d2l-button-toggle-grouped) {
+					--d2l-button-toggle-border-width: 0 1px 1px 1px;
+				}
+				div ::slotted(.d2l-button-toggle-grouped:first-child),
+				:host([dir="rtl"]) div ::slotted(.d2l-button-toggle-grouped:last-child) {
+					--d2l-button-toggle-border-radius: 0.3rem 0 0 0.3rem;
+					--d2l-button-toggle-border-width: 1px;
+				}
+				div ::slotted(.d2l-button-toggle-grouped:last-child),
+				:host([dir="rtl"]) div ::slotted(.d2l-button-toggle-grouped:first-child) {
+					--d2l-button-toggle-border-radius: 0 0.3rem 0.3rem 0;
+				}
+				:host([orientation="vertical"]) div ::slotted(.d2l-button-toggle-grouped:first-child) {
+					--d2l-button-toggle-border-radius: 0.3rem 0.3rem 0 0;
+					--d2l-button-toggle-border-width: 1px;
+				}
+				:host([orientation="vertical"]) div ::slotted(.d2l-button-toggle-grouped:last-child) {
+					--d2l-button-toggle-border-radius: 0 0 0.3rem 0.3rem;
+				}
+				div {
+					display: inline-flex;
+					flex-wrap: nowrap;
+					outline: none;
+				}
+				:host([orientation="vertical"]) div {
+					flex-direction: column;
+				}
+			</style>
+			<div aria-role="toolbar" aria-label="[[label]]">
+				<slot></slot>
+			</div>
+		`;
+	}
+	static get properties() {
+		return {
+			disabled: {
+				type: Boolean,
+				value: false,
+				observer: '_disabledChanged'
+			},
+			label: {
+				type: String,
+				value: false
+			},
+			orientation: {
+				type: String,
+				value: 'horizontal'
+			},
```

Comment:
If we don't need to support the vertical orientation I'd leave it out, as it dramatically complicates the CSS and all the keyboard stuff.